packages/dockerCompose/src/editor.ts

```diff
@@ -327,6 +327,9 @@ export class ComposeEditor {
         !Array.isArray(service.environment)
       )
         service.environment = stringifyEnvironment(service.environment);
+
+      // Remove hardcoded DNS
+      service.dns = undefined;
```

Comment:
We must check that this does not cause any trouble. Also, we should make sure the compose files of the packages are created without the `dns` instruction